src/pages/cycles/[id].js

```diff
@@ -14,7 +20,12 @@ const CyclePage = () => {
   if (process.env.NODE_ENV === 'development') {
     return (
       <>
-        <CycleSection cycleId={cycleId} />
+        <CycleSection
+          cycleId={cycleId}
+          handleLearningUnitRedirection={(learningUnitId, isCompleted) =>
+            handleLearningUnitRedirection(learningUnitId, isCompleted)
```

Comment:
it's sufficient to use:
handleLearningUnitRedirection={handleLearningUnitRedirection}